server/package-lock.json

```diff
@@ -12,7 +12,7 @@
         "axios": "^0.27.2",
         "cors": "^2.8.5",
         "dotenv": "^16.0.1",
-        "express": "^4.18.1",
+        "express": "^4.18.2",
```

Comment:
do we need this file server/package-lock.json to be pushed?

Reply:
The dev dependencies are update so latest version is addd in this lock file

Reply:
this latest version is added in the package.json file right? why needed this package-lock.json that is locally generated for that package.json right

Reply:
I have removed this file